pkg/relayer/session/claim.go

```diff
@@ -330,10 +348,34 @@ func (rs *relayerSessionsManager) payableProofsSessionTrees(
 		// Supplier CAN afford to claim the session.
 		// Add it to the claimableSessionTrees slice.
 		supplierCanAffordClaimAndProofFees := supplierOperatorBalanceCoin.IsGTE(claimAndProofSubmissionCost)
-		if supplierCanAffordClaimAndProofFees {
+
+		claimLogger := logger.With(
```

Comment:
Here and the code in line 362.

I genuinely don't think updating the claim metadata with these is useful:
- session_id
- estimated...
- claim_reward

A one-time log containing them is more than enough.

Unless we're doing data slice-and-dice on them, reading it "in english" is easier.